### PR TITLE
rg-220: template customize

### PR DESCRIPTION
**Pull request by MaxPoplavskiy**
image

image

image

image

### CODE REVIEW

File: frontend/src/bundles/edit-template/edit-template.tsx [hunk 1]
```diff
@@ -9,30 +9,74 @@ import { useParams } from 'react-router-dom';
...
+import {
+    changeBlockItemsStyle,
+    changeBlockStyle,
+    changeFontStyle,
+} from './helpers/block-styles.helper';
 import { editTemplate, getTemplateById } from './store/actions';
 import { actions as editTemplateActions } from './store/slice';
 import templateStyles from './styles.module.scss';
 
+const dropdownOptions = [
```
Comment (honkster-nazarii) [inline frontend/src/bundles/edit-template/edit-template.tsx:48]: can be moved to a separate file

**Review by honkster-nazarii — CHANGES_REQUESTED** [review]

**Comment by honkster-nazarii** [comment]
add scrolling for right panel

**Review by Valentyn13 — APPROVED** [review]

**Review by jurok3x — APPROVED** [review]

**Review by RevolveStrm — APPROVED** [review]

**Comment by siefimov** [comment]
Conflicts.

**Review by siefimov — COMMENTED** [review]
Conflicts.

**Review by honkster-nazarii — APPROVED** [review]

**Comment by MaxPoplavskiy** [comment]
[Template customize](https://trello.com/c/f3IsPTG8/220-template-customize)

**Review by Icerock10 — APPROVED** [review]

**Pull request merged**